util/src/main/DocTaglet.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2014 MongoDB, Inc.
+ * Copyright 20017 MongoDB, Inc.
```

Comment:
nit: 20017 :)

Reply:
That's probably an enforceable copyright.